.github/workflows/entropy-check.yml

```diff
@@ -0,0 +1,43 @@
+---
+name: entropy-check
+
+on:
+  pull_request_target:
+    branches:
+      - main
```

Comment:
I personally like to add the `workflow_dispatch:` trigger to workflows, so that I can launch them manually (for testing, etc.) rather than having to create a condition where they run.